### PR TITLE
Betweenness Centrality and its test

### PR DESCRIPTION
**Pull request by annacanal**
#108 Following the Kintali algorithm used in BCT

### CODE REVIEW

**Comment by annacanal** [comment]
I updated the betweenness centrality with the settings/varargin change in the constructor.
I think now formatting and indentation are correct also ^^
@giovannivolpe

File: braph/graph/measures/BetweennessCentrality.m [hunk 1]
```diff
@@ -73,17 +74,17 @@
                         for v = V
                             Q(q) = v; 
                             q = q-1;
-                            neighbours = find(G(v,:));  % neighbours of v
+                            neighbours = find(G(v, :));  % neighbours of v
```
Comment (giovannivolpe) [inline braph/graph/measures/BetweennessCentrality.m]: this is an exception, use find(G(v,:))

File: braph/graph/measures/BetweennessCentrality.m [hunk 2]
```diff
@@ -73,17 +74,17 @@
...
                             for neighbour = neighbours
-                                Duw = D(v)+ G(v,neighbour);  % Duw path length to be tested
+                                Duw = D(v)+ G(v, neighbour);  % Duw path length to be tested
```
Comment (giovannivolpe) [inline braph/graph/measures/BetweennessCentrality.m]: D(v) + G(v,neighbour)

File: braph/graph/measures/BetweennessCentrality.m [hunk 3]
```diff
@@ -73,17 +74,17 @@
...
                                 if Duw<D(neighbour)  % if new u->w shorter than old
                                     D(neighbour) = Duw;
                                     NP(neighbour) = NP(v);  % NP(u->w) = NP of new path
-                                    P(neighbour,:) = 0;
-                                    P(neighbour,v) = 1;  % v is the only predecessor
+                                    P(neighbour, :) = 0;
```
Comment (giovannivolpe) [inline braph/graph/measures/BetweennessCentrality.m]: P(neighbour,:)

File: braph/graph/measures/BetweennessCentrality.m [hunk 4]
```diff
@@ -96,10 +97,10 @@
                     end
                     
                     % Compute dependencies (DP)
-                    DP = zeros(N,1);  
+                    DP = zeros(N, 1);  
                     for n = Q(1:N-1)           
                         bc(n) = bc(n)+DP(n);  % update betweenness 
-                        for v = find(P(n,:))
+                        for v = find(P(n, :))
```
Comment (giovannivolpe) [inline braph/graph/measures/BetweennessCentrality.m]: find(P(n,:))

File: braph/graph/measures/BetweennessCentrality.m [hunk 5]
```diff
@@ -0,0 +1,153 @@
+classdef BetweennessCentrality < Measure
+    methods
+        function m = BetweennessCentrality(g, varargin)
+            m = m@Measure(g, varargin{:});
+        end
+    end
+    methods (Access=protected)
+        function bc = calculate(m)
```
[the remark this answers is not in the document]
Comment (giovannivolpe) [reply braph/graph/measures/BetweennessCentrality.m]: change the name of the output for consistency with other measures

File: braph/graph/measures/BetweennessCentrality.m (hunk 2 above)
Comment (giovannivolpe) [reply braph/graph/measures/BetweennessCentrality.m]: Add space:
D(v) + G(v, neighbour)

**Pull request merged**